bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/EnergyInfoDayAndWeek.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
```

Comment:
This should fix the build: 

```suggestion
 * Copyright (c) 2010-2021 Contributors to the openHAB project
```